util/include/rusefi/rusefi_time_types.h

```diff
@@ -21,10 +22,14 @@
  */
 using efitick_t = int64_t;
 
+using efidurus_t = std::chrono::duration<uint64_t, std::micro>;
+static_assert(sizeof(efidurus_t) == 8);
+
 /**
  * 64 bit time in microseconds (1/1_000_000 of a second), since boot
  */
-using efitimeus_t = int64_t;
+using efitimeus_t = std::chrono::time_point<std::chrono::system_clock, efidurus_t>;
```

Comment:
it probably doesn't matter, but `steady_clock` is more correct

Reply:
changed to steady_clock